firestore-genai-chatbot/README.md

```diff
@@ -73,7 +73,7 @@ Additionally, this extension uses the Google AI Gemini API. For more details on
 
 **Configuration Parameters:**
 
-* Gemini API Provider: This extension makes use of the Gemini family of large language models. Currently the extension only supports the Google AI API (for developers) but in future will support the Vertex AI Gemini API.
+* Gemini API Provider: This extension makes use of the Gemini family of large language models. For Google AI you will require an API key, whereas Vertex AI will authenticate using application default credentials.
```

Comment:
Could add a resource for those not familiar with ADC, i.e. https://firebase.google.com/docs/admin/setup#initialize-sdk